src/app/ApplicationTemplate.Tests/TestBase.cs

```diff
@@ -34,11 +35,17 @@ public virtual async Task InitializeAsync()
 			await _coreStartup.Start();
 		}
 
-		/// <summary>
-		/// A chance to configure the <paramref name="host"/> after its default configuration.
-		/// </summary>
-		/// <param name="host">Host builder</param>
-		protected virtual void ConfigureHost(IHostBuilder host) { }
+		private void AddThreadConfiguration(IHostBuilder host)
```

Comment:
If this is always required, we could put it in the CoreStartup class (perharps with `#if UNIT_TESTS` or something)

Reply:
Actually the reason I had to do this is because the place when all thread code related are created is at ViewConfiguration which it's not called during tests. Fortunately, I've found a middle ground in between both solution. I tested it quickly on one of the tests class and it seems to work. By worked, I mean that I've made sure that if we override ConfigureHost on one of the test, it still keeps IScheduler and IDispatcherScheduler registered.